test/display.test.ts

```diff
@@ -313,3 +314,65 @@ describe('display', () => {
     }
   });
 });
+
+describe('displayDependecies', () => {
+  it('should accept ID with mixed casing', async () => {
+    const depGraph: DepGraph = createFromJSON({
+      schemaVersion: '1.2.0',
+      pkgManager: { name: 'cpp' },
+      pkgs: [
+        {
+          id: 'root-node@0.0.0',
+          info: { name: 'root-node', version: '0.0.0' },
+        },
+        {
+          id: 'rOoT@1.0.0',
+          info: { name: 'rOoT', version: '1.0.0' },
+        },
+      ],
+      graph: {
+        rootNodeId: 'root-node',
+        nodes: [
+          {
+            nodeId: 'root-node',
+            pkgId: 'root-node@0.0.0',
+            deps: [
+              {
+                nodeId: 'rOoT@1.0.0',
+              },
+            ],
+          },
+          {
+            nodeId: 'rOoT@1.0.0',
+            pkgId: 'rOoT@1.0.0',
+            deps: [],
+          },
+        ],
+      },
+    });
+
+    const expected: string[] = [
+      '\nDependencies:\n',
+      '\n  rOoT@1.0.0',
+      '  confidence: 1.000',
+      '  matching files:',
+      '    - proj/foo.c',
+      '',
+    ];
+
+    const depsFilePaths: any = { 'rOoT@1.0.0': ['proj/foo.c'] };
```

Comment:
should not this be either depsFilePaths or fileSignaturesDetails containing filePaths? Maybe would be interesting to say that this test case do not consider `VulnDB` and new facts (i.e. fileSignaturesDetails) format to avoid confusion 🤔

Reply:
Both of these variables is involved in this test.
Since depsFilePaths resolves "broken --print-dep-paths"
and fileSignatureDetails resolves "missing confidence level for some packages in the CLI".

Reply:
@dekelund just so you know.. fileSignatureDetails contains filePaths (replacement of) when the ff that enables vulndb is up.

Reply:
@anthogez Thanks for the feedback. Yep, we want to make use of that, but it's not used in the CLI right now as I recall it. So, for now, in the test suite, we need to stick with the old usage, and we can update it later when the FF is respected in another PR. (Please correct me if this is not the case)

Edit: We need another PR to introduce new extraction of paths. [This](https://github.com/snyk/snyk-cpp-plugin/blob/master/lib/display/display.ts#L60) is how the existing solution works